internal/agent/device/applications/lifecycle/compose.go

```diff
@@ -38,6 +36,10 @@ func (c *Compose) add(ctx context.Context, action *Action) error {
 	projectName := action.ID
 	c.log.Debugf("Starting application: %s projectName: %s path: %s", appName, projectName, action.Path)
 
+	if err := c.ensurePodmanVolumes(ctx, action.Volumes, appName); err != nil {
```

Comment:
note: adding this here makes the status observation story of volumes align with containers.